jni_gen/examples/pdfbox_plugin/pubspec.yaml

```diff
@@ -0,0 +1,39 @@
+name: pdfbox_plugin
+description: |
+  Example of using jni_gen to generate bindings for a non-trivial library
+version: 0.0.1
+publish_to: none
+homepage: https://github.com/dart-lang/jni_gen
+
+environment:
+  sdk: ">=2.17.6 <3.0.0"
+  #flutter: ">=2.11.0"
+
+dependencies:
+  jni:
+    path: ../../../jni/
+  #flutter:
+    #sdk: flutter
+  plugin_platform_interface: ^2.0.2
+
+dev_dependencies:
+  ## Path dependency for sake of the example
+  jni_gen:
+    path: ../../
+  flutter_test:
```

Comment:
```
jni_gen/examples/pdfbox_plugin$ dart run jni_gen --config jnigen.yaml
Resolving dependencies in /usr/local/google/home/dacoharkes/src/dart-lang/jni_gen/jni_gen/examples/pdfbox_plugin... 
Because pdfbox_plugin depends on flutter_test from sdk which doesn't exist (the Flutter SDK is not available), version solving failed.
```

Reply:
I can't realistically test this because I have flutter installed also. I guess.

I will try removing flutter_test.

Btw can you post any clear examples on how to create a hybrid plugin? I am just trying to comment out flutter-specific lines now.

Reply:
is `plugin_platform_interface` also flutter-specific?